src/common/components/clusterConfiguration/DiscoveryTroubleshootingModal.tsx

```diff
@@ -22,10 +24,11 @@ type DiscoveryTroubleshootingModalProps = {
 
 export const HostsNotShowingLink: React.FC<HostsNotShowingLinkProps> = ({
   setDiscoveryHintModalOpen,
+  isSNO = false,
 }) => (
   <Button variant={ButtonVariant.link} onClick={() => setDiscoveryHintModalOpen(true)} isInline>
     <InfoCircleIcon size="sm" />
-    &nbsp;Hosts not showing up?
+    &nbsp;{pluralize(+isSNO, 'Host | Hosts')} not showing up?
```

Comment:
Did you mean
```suggestion
    &nbsp;{pluralize(+isSNO, 'Host', 'Hosts')} not showing up?
```

The third argument can be omitted as the pluralization defaults to adding 's' at the end.

Reply:
Yes, I had the wrong syntax. Also I am now using the default as from your suggestion :+1: